docs/development/getting-started.rst

```diff
@@ -299,6 +299,17 @@ This typically occur when Docker is not allocated enough memory to perform the
 migrations. Try modifying your Docker configuration to allow more RAM for each
 container and run ``make initdb`` again.
 
+
+``make initdb`` complains about PostgreSQL Version
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+You built a Warehouse install some time ago and PostgreSQL has been updated.
+If you do not need the data in your databases, it might be best to just blow
+away your builds + ``docker`` containers and start again:
+``make purge``
+``docker volume rm $(docker volume ls -q --filter dangling=true)``
```

Comment:
I think these two lines can be as simple as `docker-compose pull` to update the images for all given containers.

However it'd probably be best to just add this command as part of the `make serve` command so in the future folks don't get bit by this and have to go through the process of finding this tip.

Reply:
`docker-compose pull` won't update the contents of the persistent postgres volume, nor clean distribution static files etc., from the repo, so the `make purge && docker volume rm` command above seems right.  And given that this is far more drastic an action than `docker-compose pull`, I don't think it should be done every time anybody runs `make serve`.